person.rb

```diff
@@ -0,0 +1,23 @@
+class Person
+  attr_accessor :age
+
```

Comment:
- [ ] As requested by the previous reviewer, make sure that `:name` and `:age` instance variables should be able to be modified so they should be defined as attr_accessors (here both Getters and Setters are defined). But now you have only the `:age` instance variable as attr_accessors.
- [ ] Please make sure that the `id` is able to be retrieved so defined the `id` as Getter *(`attr_reader`) :pray: